### PR TITLE
Adds Badging Section to Killer Marmot

### PR DESCRIPTION
**Pull request by fallaciousreasoning**
Notes
- This section will only show up if `window.Badge` exists.
- In my ideal world, I'd hide this section if we weren't installed but that seems not trivial to detect?

@benfredwells Is your repo or Matt's considered master?

### CODE REVIEW

**Comment by dominickng** [comment]
This one is considered master. :)

Question: does this need an origin trial token? Should we wait until we get a token and add that this this PR?

**Comment by fallaciousreasoning** [comment]
It does yeah, I'm waiting for the trial to be properly set up before I add it. (sorry, should have marked this as WIP)

**Comment by fallaciousreasoning** [comment]
I've added the origin trial token (the trial was turned on yesterday), so I think this is ready to be reviewed :)

File: static/index.js [hunk 1]
```diff
@@ -144,4 +144,38 @@ window.addEventListener('load', e => {
...
+  setupBadgeSection();
 });
+
+const setupBadgeSection = () => {
+  const badgeSection = document.querySelector('#badge_section');
+
+  if (!badgeSection) {
+    return;
+  }
+  // Check the API is available.
```
Comment (dominickng) [inline static/index.js]: Check if the API is available.

File: static/index.js [hunk 2]
```diff
@@ -144,4 +144,38 @@ window.addEventListener('load', e => {
...
+  if (!window.Badge) {
```
Comment (dominickng) [inline static/index.js]: ExperimentalBadge? Here and elsewhere.

Comment (fallaciousreasoning) [reply static/index.js]: Oooh now that was lucky :D

File: static/index.js (hunk 1 above)
Comment (fallaciousreasoning) [reply static/index.js]: Changed to "Check that the API is available"

File: static/index.js (hunk 2 above)
Comment (fallaciousreasoning) [reply static/index.js]: If I use 
```js
const badgeAPI = window.Badge || window.ExperimentalBadge;
```
would that be assuming too much?

**Review by dominickng — APPROVED** [review]
Thanks! LGTM

**Pull request merged**